components/style/gecko_bindings/mod.rs

```diff
@@ -2,24 +2,5 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
```

Comment:
Moved the script back to the same folder bindings.rs is, added the generated file there, and removed the changes to files that should not have been changed.